#! /bin/sh /usr/share/dpatch/dpatch-run
## 567580_menu_prompt_reflects_program.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: It has been requested, that gnupg shows a less ambiguous prompt than
## DP: "Command>". Upstream agreed.
## DP:
## DP: <URL:http://bugs.debian.org/567580>
## DP: <URL:https://bugs.g10code.com/gnupg/issue1187>
## DP: <URL:http://lists.gnupg.org/pipermail/gnupg-commits/2010-February/008766.html>

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' trunk~/g10/card-util.c trunk/g10/card-util.c
--- trunk~/g10/card-util.c	2009-08-13 09:14:37.000000000 +0200
+++ trunk/g10/card-util.c	2010-03-21 14:45:32.355740548 +0100
@@ -1817,7 +1817,7 @@
 	    if (!have_commands)
               {
 		tty_enable_completion (card_edit_completion);
-		answer = cpr_get_no_help("cardedit.prompt", _("Command> "));
+		answer = cpr_get_no_help("cardedit.prompt", _("gpg/card> "));
 		cpr_kill_prompt();
 		tty_disable_completion ();
               }
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' trunk~/g10/keyedit.c trunk/g10/keyedit.c
--- trunk~/g10/keyedit.c	2009-08-26 07:35:31.000000000 +0200
+++ trunk/g10/keyedit.c	2010-03-21 14:45:32.355740548 +0100
@@ -1616,7 +1616,7 @@
 	    if( !have_commands )
 	      {
 		tty_enable_completion(keyedit_completion);
-		answer = cpr_get_no_help("keyedit.prompt", _("Command> "));
+		answer = cpr_get_no_help("keyedit.prompt", "gpg> ");
 		cpr_kill_prompt();
 		tty_disable_completion();
 	      }
